### ISSUE TITLE
Investigate failing (and extra) request to Maps API

### ISSUE THREAD

**Issue by alonsogarciapablo** [CartoDB/cartodb.js#1225]
Vizualizations with a CartoDB Layer and a Torque Layer are triggering two requests to the Maps API and the first one is missing the `cartocss` option for the torque layer:

```
{
  "layers": [
    {
      "type": "cartodb",
      "options": {
        "sql": "select * from twitter_t3chfest_reduced",
        "cartocss": "/** simple visualization */\n\n#twitter_t3chfest_reduced{\n  marker-fill-opacity: 0.9;\n  marker-line-color: #FFF;\n  marker-line-width: 1;\n  marker-line-opacity: 1;\n  marker-placement: point;\n  marker-type: ellipse;\n  marker-width: 10;\n  marker-fill: #FF6600;\n  marker-allow-overlap: true;\n}",
        "cartocss_version": "2.1.1",
        "interactivity": [],
        "widgets": {}
      }
    },
    {
      "type": "torque",
      "options": {
        "cartocss_version": "2.1.0",
        "interactivity": [],
        "widgets": {}
      }
    }
  ]
}
```

Following viz.json fails for example:
https://juanignaciosl-ded02.cartodb-staging.com/api/v3/viz/ade79050-fbd8-11e5-9aac-04013fc66a01/viz.json

cc: @juanignaciosl

**Comment by juanignaciosl** [comment]
The missing torque attributes are fixed at CartoDB/cartodb#7051.

**Comment by alonsogarciapablo** [comment]
Closing this as it seems to be working now.
